apps/client/components/navbar/Navbar.tsx

```diff
@@ -12,14 +12,30 @@ import { NavLink } from './NavLink'
 
 export const Navbar = () => {
   const [isOpenedNav, setIsOpenedNav] = useState(false)
+  const [isLower, setIsLower] = useState(false)
+  const navbarRef: any = useRef()
```

Comment:
refa typuje się tak useRef<HTMLDivElement | null>(null(
gdzie HTMLDivElement możesz zmieniać w zależności czym ten ref jest
W przypadku nava będzie to po prostu ogólnie HTMLElement, ale dużo elementów ma bardziej dokładne typy.